.travis.yml

```diff
@@ -4,6 +4,7 @@ install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.t
 script: bash -ex .travis-opam.sh
 env:
   global:
+  - EXTRA_REMOTES="https://github.com/mirage/mirage-dev.git"
```

Comment:
hmm, this should not land in master, or?

Reply:
An advantage of this is that I can tag, release and then remove this line keeping the CI green throughout. Without this line the CI goes red (but obviously we could choose to ignore that)